cylc/flow/cfgspec/globalcfg.py

```diff
@@ -939,6 +934,10 @@
                  * ``background``
                  * ``slurm``
                  *  ``pbs``
+
+                .. seealso::
+
+                   :ref:`List of built-in Job Runners <tutorial-job-runner>`
```

Comment:
`<AvailableMethods>` is a better reference? (although could do with improving)

Reply:
```suggestion
                    :ref:`List of built-in Job Runners <AvailableMethods>`
```